R/test.R

```diff
@@ -229,3 +229,59 @@ NULL
 #' @keywords internal
 #' @family tests
 NULL
+
+#' Test case: section highlighting
```

Comment:
I think we also need a couple of explicit tests for the new behaviour, since it'll be so easy to break it by accident.